http/vertx-web-client/src/test/java/io/quarkus/qe/vertx/webclient/ChuckNorrisResourceIT.java

```diff
@@ -0,0 +1,197 @@
+package io.quarkus.qe.vertx.webclient;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.get;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlEqualTo;
+import static io.restassured.RestAssured.given;
+import static org.awaitility.Awaitility.await;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.empty;
+import static org.hamcrest.Matchers.greaterThan;
+import static org.hamcrest.Matchers.not;
+import static org.hamcrest.core.Every.everyItem;
+import static org.hamcrest.core.StringContains.containsString;
+import static org.hamcrest.text.IsEqualIgnoringCase.equalToIgnoringCase;
+
+import java.io.File;
+import java.net.HttpURLConnection;
+import java.nio.file.Paths;
+import java.time.Duration;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.http.HttpStatus;
+import org.hamcrest.Matcher;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.qe.vertx.webclient.resources.WireMockChuckNorrisResource;
+import io.quarkus.test.bootstrap.JaegerService;
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.scenarios.annotations.EnabledOnNative;
+import io.quarkus.test.services.JaegerContainer;
+import io.quarkus.test.services.QuarkusApplication;
+import io.restassured.response.Response;
+
+@QuarkusScenario
+public class ChuckNorrisResourceIT {
+    static final String EXPECTED_ID = "aBanNLDwR-SAz7iMHuCiyw";
+    static final String EXPECTED_VALUE = "Chuck Norris has already been to mars; that why there's no signs of life";
+    static final int DELAY = 3500; // must be greater than vertx.webclient.timeout-sec
+
+    private static final int REST_PORT = 16686;
+    private static final int TRACE_PORT = 14250;
+
+    private Response resp;
+
+    @JaegerContainer(image = "jaegertracing/all-in-one:latest", tracePort = TRACE_PORT, restPort = REST_PORT, expectedLog = "\"Health Check state change\",\"status\":\"ready\"")
+    static final JaegerService jaeger = new JaegerService();
+    private final String jaegerEndpoint = jaeger.getRestUrl();
+
+    @QuarkusApplication
+    static WireMockChuckNorrisResource STUB = new WireMockChuckNorrisResource()
```

Comment:
I don't understand this ... what you want is a wiremock server, not a Quarkus application, right?

Reply:
Yes. Is there a way to start the BaseService without using this annotation?

Reply:
You can use either:
- Deploy wiremock using containers: use `@Container` annotation. This would make the test compatible to deploy it in OCP
- Do what you're doing in WireMockChuckNorrisResource (start a HTTP server in your local machine), directly in the test case using the `vertx.onPreStart()` and stopping the HTTP server in the `vertx.onPostStart()` or in the `@AfterAll` junit5 stage. Obviously, this approach would make the test case incompatible with deploying it in OCP.

Reply:
Changed this to use WireMock container